lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/impl/DefaultRemoteManagerImplementation.java

```diff
@@ -47,7 +49,7 @@
 
   private static final Logger LOG = Logger.getLogger(HandlerContainer.class.getName());
 
-  private static final AtomicInteger COUNTER = new AtomicInteger(0);
+  private static final AtomicInteger counter = new AtomicInteger(0);
```

Comment:
Yes, this will violate ConstantName check (static final field => must be ALL_CAPS). Currently this check is warning-level, not error-level, because we still have some misnamed constants which are public (so can't be renamed easily, have to do deprecation process). I will go through such violations and add exceptions for them so that we can enforce all checks in Naming category and stop introducing new violations